pkg/cmd/agentversionjob.go

```diff
@@ -75,12 +75,15 @@ func (avj *AgentVersionCheckJob) Execute() error {
 	err = avj.getJFrogVersions(avj.urlName)
 	if err != nil {
 		log.Trace(err)
-		return err
-	}
-	// compare build to latest version
-	if isVersionStringOlder(avj.buildVersion, config.AgentLatestVersion) {
-		log.Warnf("New version available. Please consider upgrading from version %s to version %s", avj.buildVersion, config.AgentLatestVersion)
+		// Could not get update from jfrog.  Warn that we could not determine version and continue processing
+		log.Warn("Agent cannot determine the next available release. Be aware that your agent could be outdated.")
```

Comment:
I like "out of date" rather than "outdated", but not a big deal either way.

Reply:
@dfeldick i'll check with C...  I mean, I say that milk could be outdated.  ;)